### DISCUSSION TITLE
Adding possebillity to change FFP-Module output to hourly results instead of only offering aggregated results

### DISCUSSION THREAD

**Discussion by 030Skript** [UMEP-dev/UMEP#641] [Ideas]
Hello everyone,

Unfortunately I just ran the FFP module without checking the output options carefully. So I've just realised that I actually need hourly values to compare my fc data (as was my input), rather than the aggregated result. I suspect I'm not the only one, so it would be nice if it were possible to add this feature for the Kljun (2015) FFP, or am I missing something? Or is it somehow possible to revert the output back to hourly resolution? 

Best regards from Berlin

**Comment by biglimp** [comment]
Hi,
No, unfortunately this tool only produce aggregated footprints and to change this would require some hands on coding. We also did not have the time to migrate the tool to UMEP for Processing where your wishes would be much easier to complete using a small python script.
